package.json

```
@@ -1498,12 +1498,12 @@
                 {
                     "command": "aws.cwl.searchLogGroup",
                     "group": "0@1",
-                    "when": "view == aws.explorer && viewItem == awsCloudWatchLogNode"
+                    "when": "view == aws.explorer && viewItem == awsCloudWatchLogNode || view == aws.explorer && viewItem == awsCloudWatchLogParentNode"
```

You can do regular expressions with `~=`:
```suggestion
                    "when": "view == aws.explorer && viewItem ~= awsCloudWatchLogNode|awsCloudWatchLogParentNode"
```